src/core/Jalasoft.TeamUp.Projects.Core/ProjectsService.cs

```diff
@@ -46,9 +46,13 @@ public Project PostProject(Project project)
                 var result = this.projectsRepository.Add(project);
                 return result;
             }
+            catch (ValidationException exVal)
+            {
+                throw new ProjectsException(ProjectsException.ProjectsErrors.BadRequest, exVal);
```

Comment:
again, the service doesn't know anything about HTTP language (not found, bad request, internal server error) those terms should be used at API layer only

Reply:
done